samples/sensor/accel_polling/src/main.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2022 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <drivers/sensor.h>
+#include <zephyr.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+static const struct device *sensors[] = {
+#if DT_NODE_EXISTS(DT_ALIAS(accel0))
+	DEVICE_DT_GET(DT_ALIAS(accel0)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel1))
+	DEVICE_DT_GET(DT_ALIAS(accel1)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel2))
+	DEVICE_DT_GET(DT_ALIAS(accel2)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel3))
+	DEVICE_DT_GET(DT_ALIAS(accel3)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel4))
+	DEVICE_DT_GET(DT_ALIAS(accel4)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel5))
+	DEVICE_DT_GET(DT_ALIAS(accel5)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel6))
+	DEVICE_DT_GET(DT_ALIAS(accel6)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel7))
+	DEVICE_DT_GET(DT_ALIAS(accel7)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel8))
+	DEVICE_DT_GET(DT_ALIAS(accel8)),
+#endif
+#if DT_NODE_EXISTS(DT_ALIAS(accel9))
+	DEVICE_DT_GET(DT_ALIAS(accel9)),
+#endif
+};
+
+static const enum sensor_channel channels[] = {
+	SENSOR_CHAN_ACCEL_X,
+	SENSOR_CHAN_ACCEL_Y,
+	SENSOR_CHAN_ACCEL_Z,
+};
+
+void format_sensor_value(struct sensor_value *value, char *buf, size_t len)
+{
+	if (value->val1 == 0) {
+		snprintf(buf, len, "  %c0.%06u", (value->val2 < 0 ? '-' : ' '), abs(value->val2));
+		return;
+	}
+	snprintf(buf, len, "%4d.%06u", value->val1, abs(value->val2));
+}
```

Comment:
why not use `sensor_value_to_double` and print the value?

Reply:
No strong reason is there.
Fix to use sensor_value_to_double.